app/views/editor/wikiRich.html.erb

```diff
@@ -128,6 +144,10 @@
 
 
 <script>
+  $(window).on('load',function(){
```

Comment:
Hi! My apologies for the slowness here. Could we move this up into the same block of code as above, so it's all driven from the same place? Let me try making a code suggestion. Thanks!